docs/CONTRIBUTING.md

```diff
@@ -14,13 +14,27 @@ If you are a *git ninja*, we feel happy and sad for you at the same time. Your s
 
-## Pull request guidelines
+## Pull request submission guidelines
 * **Create topic branches**. Never ever open a pull request from your master branch. **Ever!**
 * **Code against master branch**. Unless release specific, all Kodi development occurs in the `master` branch. If it needs backporting it can be done after it hits master.
 * **One topic branch per change**. Found something that needs fixing but is unrelated to the current work? Create a new topic branch and open another *pull request*.
 * **Use a coherent commit history**. Make sure each individual commit in your *pull request* is meaningful and organized in logical chunks. Tidy up and squash commits before submitting.
-* **Be descriptive, but concise**. The *pull request* and commits should have a useful title and follow the `[component(s)] Short description...` format. No, `[README.md] Update` is **not** descriptive enough. The *pull request* description should only contain information relevant to the change.
-* **Be patient, be kind, rewind**. We're all volunteers with busy lives but usually very responsive in cataloging *pull requests* and pinging the relevant maintainer(s). Sometimes though, things go unnoticed for longer than usual. Probably due to beer deprivation... or lack thereof. Feel free to nudge us if we forget.
+* **Be descriptive, but concise**. The *pull request* and *commits* should have an useful title and follow the format:<br/>
+`[Component(s)] Short description...` or alternative format `Component(s): Short description...`<br/>
+by component we mean for example, identify the area of the code (gui, video, depends, builds, etc), the code context (e.g. class name), etc... you can easily find hints by browsing through the git history, an example:<br/>
+`[Python][xbmcgui] Fix x problem...` or `Python/xbmcgui: Fix x problem...`<br/>
+instead of something like `[README.md] Update` is **not** descriptive enough.<br/>
+Also adding a short description to the *commit* (under the title) is a good thing to consider to give a better explanation of the commit changes.<br/>
+The *pull request* description should only contain information relevant to the change.
+* **Draft PR**. If your work requires further development, and you wish to ask the opinion of other developers on how to proceed, you can publish your PR as Draft.
+
+## Pull request merge guidelines
+* **First rule, be patient, be kind, rewind**. We're all volunteers with busy lives but usually very responsive in cataloging *pull requests* and pinging the relevant maintainer(s).
+* **Jenkins code formatting requests**. Always apply code style formatting requests from Jenkins. If not, please provide a valid reason why you do not apply them.
+* **Jenkins CI builds**. It is mandatory to have green light on all Jenkins building platforms. Set "No-Jenkins" flag if your pull request is yet not complete or you do not want to have jenkins building the PR code.
```

Comment:
```suggestion
* **Jenkins CI builds**. It is mandatory to have green light on all Jenkins building platforms. The "No-Jenkins" label can be set by maintainers to avoid having the CI building the PR code (and wasting build resources) in cases the contribution is still work in progress or if known in advance it won't build.
```